src/components/BigNumber.js

```diff
@@ -0,0 +1,15 @@
+import PropTypes from 'prop-types';
+
+import { getUnitPrefix } from '@/helpers/numbers';
+
+const BigNumber = ({ children, unit }) => {
+  const { number, prefix } = getUnitPrefix(children);
```

Comment:
Since you return prefix AND number this should rather be named 'transformBigNumber' or similar to make clear what this helper is actually doing.